guide/index.md

```diff
@@ -6,81 +6,218 @@ sidebar_position: 1
 
 Integrate Fishjam Cloud into your React Native application.
 
+:::info
+This guide assumes you're using Expo Bare Workflow
+:::
+
 ### Install package
 
 ```bash npm2yarn
 npm install @fishjam-cloud/react-native-client
 ```
 
-### Fetch participant token
+### Build native dependencies
+
+```bash
+npx expo prebuild --clean
+```
+
+### Get Room Manager URL
+
+Login to [Fishjam Cloud Dashboard](https://fishjam.io/app) and get your Room Manager URL.
+
+## TLDR:
+
+Checkout [complete example below:](#full-example)
 
-Login to [Fishjam Cloud Dashboard](https://fishjam.io/app) and get your Room Manager url. Next, use it to fetch
-participant token for new room:
+## Step by step instructions:
+
+### 1. Fetch peer token
+
+Use your room manager URL to fetch peer token to get a new room:
 
 ```ts
 const response = await fetch(
-  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?room=ROOM_NAME&user=USER_NAME`,
+  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/*roomName*/users/*username*`,
```

Comment:
Let's use new params: 
```suggestion
  `https://fishjam.io/api/v1/connect/*YOUR_ID*/room-manager/?roomName=*roomName*&userName=*username*`,
```

Reply:
Does it work?
I tried it yestarday and params passed like this didn't work :/

Reply:
Are you sure this is the new way? We have slashes both on staging and prod.

Reply:
slashes are now deprecated. But also we should update `participantName` back to peerName

Reply:
fixed